proto/decentraland/kernel/comms/rfc4/comms.proto

```diff
@@ -14,9 +14,45 @@ message Packet {
     Chat chat = 5;
     Scene scene = 6;
     Voice voice = 7;
+    SdkComponent sdk_component = 8;
+    CustomSignal signal = 9;
```

Comment:
_I may have gone overboard with this comment. Sorry! 😅_

I recommend we change the names to `Component` and `Custom`.

My thoughts:

#### Rationale for `Component`

1. I think we should avoid having hard references to the SDK at the protocol layer. It's fine for comments, but a variable name feels like conceptually breaking through the layers 👊 .

2. I don't think the word `Component` needs clarification. The meaning is very established in our terminology, and has been for a long while.
 
3. While `Sdk` does additionally imply that the `Component` is standard, that's a reasonable assumption of any message in the protocol unless otherwise specified.

#### Rationale for `Custom`

1. The term `Signal` is associated with semantics that don't _quite_ fit here.

2. `Custom`, removing `Signal`, matches the style of the other names (e.g. we don't have `ChatMessage` or `VoiceSample`) and is explicit about the purpose of the field -- this is for unspecified stuff.

3. I can't think of other places where we'd reuse this concept of `Signal`. If this is going to be the only case, I don't think we should introduce it.